### PR TITLE
add FeatureRenderer

### PR DESCRIPTION
**Pull request by bartvde**
This is ready for review, TIA for any review. Please note that the update function is not yet supported, but this can be added in the future.

### CODE REVIEW

File: examples/featurerenderer.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <link rel="stylesheet" href="../resources/css/gxm.css" type="text/css">
+        <link rel="stylesheet" href="./examples.css" type="text/css">
+        
+        <!-- Load Sencha Touch -->
+        <script type="text/javascript" src="http://cdn.sencha.io/touch/sencha-touch-2.0.1/sencha-touch-all-debug.js"></script>
+        <!-- Load OpenLayers -->
+        <!-- You should definitely consider using a custom single-file version of OpenLayers -->
+        <script type="text/javascript" src="http://openlayers.org/api/2.11/OpenLayers.js"></script>
+        <!-- Load OpenStreetMap -->
+        <script type="text/javascript" src="http://www.openstreetmap.org/openlayers/OpenStreetMap.js"></script>
```
Comment (marcjansen) [inline examples/featurerenderer.html]: Do we need this in this example?

File: examples/featurerenderer.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+ * text of the license.
+ */
+
+/** api: example[featurelist]
+ *  MapPanel and interacting Featurelist
+ *  ----------------
+ *  This example shows the use of the GXM.FeatureList-class to manage and 
+ *  interact with the features of a GXM.MapPanel.
+ */
+
```
Comment (marcjansen) [inline examples/featurerenderer.js]: The docs above are not correct; and currently we are not doing anything with it either. I'd suggest removing the block all together.

File: examples/featurerenderer.html (hunk 1 above)
Comment (bartvde) [inline examples/featurerenderer.html]: not needed, I'll remove it, just copied the header from other examples.

**Comment by marcjansen** [comment]
@bartvde except for the above minor comments; this is fine for master and you should merge it. 

Relevant tests pass and the example looks as expected in Android 2.3 on a SGS I.

Thanks for the work on this. And of course we will need the update-method somewhere in the future.

**Comment by marcjansen** [comment]
It'd be super cool if you could have sth. like the source btn, that the other examples have. Feel free to merge with or without this suggestion implemented.

**Pull request merged**